lib/storage/storage-init.js

```diff
@@ -48,8 +49,11 @@ function initStore(dataSetId, seedData, mediator) {
         return dataStore;
       });
     }).then(function(dataStore) {
-      //Initialisation complete, subscribe to any mediator topics
-      return dataStore.listen(dataTopicPrefix, mediator);
+      if (customListenHandler) {
```

Comment:
Not sure why this is here. Is the list override not done in https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/pull/99/files#diff-c532aa5afbd5aa381c496fff5e1b8754R41 ?